locations/spiders/albertsons.py

```diff
@@ -0,0 +1,116 @@
+import json
+import re
+import scrapy
+from locations.items import GeojsonPointItem
+
+
+class AlbertsonsSpider(scrapy.Spider):
+    name = "albertsons"
+    allowed_domains = ["locator.safeway.com"]
```

Comment:
Looks like this is pulling in stores for Safeway, not Albertson's.

Try https://local.albertsons.com/index.html as a spot to start?